tools/GraphClust/CollectResults/glob_report.xml

```diff
@@ -47,6 +47,19 @@
           $final_partition_used_cmsearch
         #end if
 
+				#if  str($advanced_opts.advanced_opts_selector) == "show":
+						#if  str($advanced_opts.param_type.param_type_selector) == "gclust":
+									$advanced_opts.param_type.p
```

Comment:
I guess it's because of the editor preferences of showing tabs with 2/4 characters. @eteriSokhoyan replacing tabs with two spaces should fix this. For future using a good editor one can configure to automatically change tabs two spaces.